python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -827,10 +829,26 @@ def convert_gelu(g, op, block):
     """Operator converter for gelu."""
 
     x = g.get_node(op.input("X")[0])
-    out = x * (
-        _expr.const(0.5, dtype="float32")
-        + _op.erf(x * _expr.const(0.5**0.5, dtype="float32")) * _expr.const(0.5, dtype="float32")
-    )
+    approximate = op.attr("approximate")
```

Comment:
There's no need to implement `approximate` strategy, it's just a strategy to boost computation in Paddle